### PR TITLE
test: improve master/workerrpc unit test coverage to 91%

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

part of #998 

- improve `master/workerrpc` unit test coverage to 91%

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: dm/master/workerrpc/workerrpc_test.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
+// Copyright 2019 PingCAP, Inc.
```
Comment (lance6716) [inline dm/master/workerrpc/workerrpc_test.go:1]:
```suggestion
// Copyright 2020 PingCAP, Inc.
```

File: dm/master/workerrpc/workerrpc_test.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	workerCli.EXPECT().SwitchRelayMaster(gomock.Any(), reqs[6].SwitchRelayMaster)
+	workerCli.EXPECT().OperateRelay(gomock.Any(), reqs[7].OperateRelay)
+	workerCli.EXPECT().PurgeRelay(gomock.Any(), reqs[8].PurgeRelay)
+	workerCli.EXPECT().UpdateRelayConfig(gomock.Any(), reqs[9].UpdateRelay)
+	workerCli.EXPECT().MigrateRelay(gomock.Any(), reqs[10].MigrateRelay)
+	workerCli.EXPECT().OperateSchema(gomock.Any(), reqs[11].OperateSchema)
+	workerCli.EXPECT().OperateV1Meta(gomock.Any(), reqs[12].OperateV1Meta)
+	workerCli.EXPECT().HandleError(gomock.Any(), reqs[13].HandleError)
+
+	for _, req := range reqs {
```
Comment (lance6716) [inline dm/master/workerrpc/workerrpc_test.go:133]: could be combined to below loop

**Review by lance6716 — COMMENTED** [review]
rest LGTM

File: dm/master/workerrpc/workerrpc_test.go (hunk 2 above)
Comment (csuzhangxc) [reply dm/master/workerrpc/workerrpc_test.go]: done in https://github.com/pingcap/dm/pull/1047/commits/d919dc5317f620f5287190df35a1bd3823e08759.

**Comment by lance6716** [comment]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**